index.js

```diff
@@ -137,7 +137,8 @@ class JestReportPortal {
 
     _finishSkippedTest (isRetried) {
         const status = 'skipped';
-        const finishTestObj = Object.assign({ status }, { retry: isRetried });
+        const issue = this.reportOptions.skippedIssue === false ? { issueType: 'NOT_ISSUE' } : null;
+        const finishTestObj = Object.assign({ status }, { retry: isRetried }, issue && { issue });
```

Comment:
You need a condition if you want to include retry just when it's true. For now there is no sense to divide `status` and `retry` into separate objects.